README.md

```diff
@@ -124,6 +124,15 @@ then config file, then environment variables.
 
 [Valid timeout units](https://pkg.go.dev/time#ParseDuration) are "ns", "us"/"µs", "ms", "s", "m", "h".
 
+### Endpoint URIs
+
+otel-cli deviates from the OTel specification for endpoint URIs.
+
+   * bare `host:port` endpoints are assumed to be gRPC
+   * `grpc://` URIs are supported
+   * `http://` and `https://` are assumed to be HTTP _only_ (the spec reuses them for gRPC)
```

Comment:
Can you please confirm that it's still about OTLP/gRPC and not OTLP/HTTP protocol (see `OTEL_EXPORTER_OTLP_PROTOCOL`).
I'm referring to the exporter specification https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options